src/tests/unit/pizzas-dao.test.js

```diff
@@ -0,0 +1,156 @@
+import chai from 'chai';
+import chaiAsPromised from 'chai-as-promised';
+import config from 'config';
+import proxyquire from 'proxyquire';
+import sinon from 'sinon';
+import sinonChai from 'sinon-chai';
+
+import { getPizzaByIdData } from './test-data';
+
+const should = chai.should();
+chai.use(chaiAsPromised);
+chai.use(sinonChai);
+
+let connectionStub;
+let getDoughByIdStub;
+let pizzasDao;
+let result;
+let resultError;
+let serializePizzaStub;
+
+const {
+  baseSerializerReturn,
+  emptyDbReturn,
+  fullRawPizza,
+  getPizzaByIdQuery,
+  getPizzaIngredientsQuery,
+  multipleRowsReturn,
+  pizzaDbReturn,
+  pizzaIngredientsDbReturn,
+  serializedDoughReturn,
+} = getPizzaByIdData;
+
+const setUpDatabaseStubs = (firstCall, secondCall, serializerReturn) => {
+  connectionStub = sinon.stub();
+  connectionStub.onCall(0).resolves(firstCall);
+  connectionStub.onCall(1).resolves(secondCall);
+  getDoughByIdStub = sinon.stub().returns(serializerReturn);
+};
+
+const proxyquirePizzasDao = () => {
+  const getConnectionStub = sinon.stub().resolves({
+    execute: connectionStub,
+    close: () => {},
+  });
+
+  return proxyquire('api/v1/db/oracledb/pizzas-dao', {
+    './connection': {
+      getConnection: getConnectionStub,
+    },
+    '../../serializers/pizzas-serializer': {
+      serializePizza: serializePizzaStub,
+    },
+    './doughs-dao': {
+      getDoughById: getDoughByIdStub,
+    },
+  });
+};
+
+
+describe('test pizzas DAO', () => {
+  before(() => {
+    sinon.replace(config, 'get', () => ({ oracledb: {} }));
+  });
+  after(() => {
+    sinon.restore();
+  });
+  context('getPizzaById', () => {
+    let inputId;
+    before(() => {
+      serializePizzaStub = sinon.stub().returns(baseSerializerReturn);
+    });
+    beforeEach(async () => {
+      pizzasDao = proxyquirePizzasDao();
+      try {
+        result = await pizzasDao.getPizzaById(inputId);
+      } catch (err) {
+        resultError = err;
+      }
+    });
+    afterEach(() => {
+      serializePizzaStub.resetHistory();
+      connectionStub.resetHistory();
+    });
+    context('when it is called with a valid id', () => {
+      before(() => {
+        inputId = '1';
+      });
```

Comment:
Any one line arrow function can be reduced like in the comment above.

Reply:
eslint doesn't like it when you return assignments but I'm not sure if that's intended to apply in this case or not 🤔 .

[Rule here](https://eslint.org/docs/rules/no-return-assign)